contracts/MultiCreateV1.sol

```diff
@@ -0,0 +1,76 @@
+pragma solidity ^0.8.7;
+import "./interfaces/IControllerV1.sol";
+import "./interfaces/IMemberToken.sol";
+
+/* solhint-disable indent */
+
+contract MultiCreateV1 {
+    IMemberToken immutable memberToken;
+
+    constructor(address _memberToken) {
+        memberToken = IMemberToken(_memberToken);
+    }
+
+    function createPods(
+        IControllerV1 _controller,
+        address[][] memory _members,
+        uint256[] calldata _thresholds,
+        address[] memory _admins,
+        bytes32[] memory _labels,
+        string[] memory _ensStrings,
+        string[] memory _imageUrls
+    ) public returns (address[] memory) {
+        uint256 numPods = _thresholds.length;
+        require(_members.length == numPods, "incorrect members array");
+        require(_labels.length == numPods, "incorrect labels array");
+        require(_ensStrings.length == numPods, "incorrect ensStrings array");
+        require(_imageUrls.length == numPods, "incorrect imageUrls array");
+
+        uint256 nextPodId = memberToken.getNextAvailablePodId();
+
+        // 1 indexing to avoid relying on address(0)
+        address[] memory newPods = new address[](numPods + 1);
+
+        for (uint256 i = 0; i < numPods; i++) {
+            // if the numerical version of admin address is less than numPods + 1 and not address(0) its a pointer
```

Comment:
I don't understand what this if statement is looking for, like what the relationship between numPods + 1 and the contents of the admin array are? What is the casting doing?